cuizinart/frontend/src/index.js

```diff
@@ -337,6 +343,20 @@ class CuizinartApp extends Component {
             });
     };
 
+    register = (email, password, re_password) => {
+        let self = this;
+        this.setState({ isLoading: true });
+        axios.post('/register', { 'email': email, 'password': password, 're_password': re_password})
```

Comment:
I might be wrong, but I don't think flask-security is expecting `re_password`?

Reply:
Let me delete re_password and test it again. Acutally this re_pwd is only used in validateForm, I guess I was just trying to reserve and pass this parameter in case it would be used.